plugin/src/functions/orchestrator/handler.ts

```diff
@@ -18,18 +25,25 @@ const orchestrator = async (event: SQSEvent) => {
           requestPayload,
         };
 
-        const response = await lambda
-          .invoke({
-            FunctionName: functionName,
-            Payload: JSON.stringify(event),
-          })
-          .promise();
+        const response = await invokeLambdaFunction(functionName, event);
 
         const finishedAt = new Date().toISOString();
 
+        const decoder = new TextDecoder('utf-8');
+
+        const payload =
+          response.Payload === undefined
+            ? {}
+            : decoder.decode(response.Payload);
+
+        console.log({
```

Comment:
?

Reply:
In aws-sdk v3 for whatever reason the payload is now base64 and isn't decoded automatically by the SDK.
We have to decode it ourself.

Reply:
I don't understand why we then have to log it?

Reply:
Oh oops I was looking at the wrong line. I've pushed a fix